HostToController.h

```diff
@@ -150,6 +177,9 @@ class HostToController : public UnidirectionalProxy<HostToController> {
         packet_state = WAITING_FOR_PACKET_TYPE;
         packet_index = 0;
         packet_length = 0;
+        /* link layer takes ownerhsiop of the WSF message */
```

Comment:
Comment and logics are wrong for drivers which are not Cordio zero copy controller.